docs/ref/databases.txt

```diff
@@ -398,11 +398,15 @@ Collation settings
 ~~~~~~~~~~~~~~~~~~
 
 The collation setting for a column controls the order in which data is sorted
-as well as what strings compare as equal. It can be set on a database-wide
-level and also per-table and per-column. This is `documented thoroughly`_ in
-the MySQL documentation. In all cases, you set the collation by directly
-manipulating the database tables; Django doesn't provide a way to set this on
-the model definition.
+as well as what strings compare as equal. You can specify the ``db_collation``
+parameter to set the collation name of the column for
+:attr:`CharField <django.db.models.CharField.db_collation>` and
+:attr:`TextField <django.db.models.TextField.db_collation>`.
+
+It can also be set on a database-wide level and per-table. This is
```

Comment:
I'm not sure `It` carries from the previous paragraph...

```suggestion
The collation can also be set on a database-wide level and per-table. This is
```

Wrapping?

Reply:
Updated.